qemu/tests/cfg/nic_opt.cfg

```diff
@@ -0,0 +1,71 @@
+- nic_opt:
+    type = nic_opt
+    only virtio_net
+    only x86_64 ppc64 ppc64le
+    # ping test setting
+    ping_package_sizes = 0 1 48 64 512 1440 1500 1505 4096 4192 32767 65507
+    # netperf test setting
+    netperf_link = netperf-2.6.0.tar.bz2
+    server_path = /var/tmp/
+    client_path = /var/tmp/
+    netperf_test_duration = 180
+    netperf_output_unit = m
+    netperf_local_cpu = yes
+    netperf_remote_cpu = yes
+    netperf_para_sessions = 1
+    test_protocols = TCP_STREAM
+    netperf_client = ${main_vm}
+    netperf_server = localhost
+    netperf_package_sizes = 1024
+    sub_type = netperf_stress
+    sub_exit_timeout = 10
```

Comment:
@sitoliu I would suggest put vhost=on here. As vhost=on is default option of virtio-net from libvirt layer.